.github/workflows/ci.yaml

```diff
@@ -34,3 +34,22 @@ jobs:
         uses: android-actions/setup-android@v3
       - name: Unit tests
         run: ./gradlew tests:test
+
+  docs:
+    name: Build Documentation
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
+        with:
+          gradle-version: "8.11.1"
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v3
+      - name: Build documentation
```

Comment:
Are we planning to publish this on PRs that contain changes? Or just building it to make sure is not breaking?